packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -203,30 +204,45 @@ gmaps.InfoWindowOptions? _infoWindowOptionsFromMarker(Marker marker) {
 
   // Add an outer wrapper to the contents of the infowindow, we need it to listen
   // to click events...
-  final HtmlElement container = DivElement()
+  final HTMLElement container = createDivElement()
     ..id = 'gmaps-marker-${marker.markerId.value}-infowindow';
 
   if (markerTitle.isNotEmpty) {
-    final HtmlElement title = HeadingElement.h3()
-      ..className = 'infowindow-title'
-      ..innerText = markerTitle;
-    container.children.add(title);
+    final HTMLHeadingElement title =
+        (document.createElement('h3') as HTMLHeadingElement)
+          ..className = 'infowindow-title'
+          ..innerText = markerTitle;
+    container.appendChild(title);
   }
   if (markerSnippet.isNotEmpty) {
-    final HtmlElement snippet = DivElement()
-      ..className = 'infowindow-snippet'
+    final HTMLElement snippet = createDivElement()
+      ..className = 'infowindow-snippet';
+
+    // Firefox and Safari don't support Trusted Types yet.
+    // See https://developer.mozilla.org/en-US/docs/Web/API/TrustedTypePolicyFactory#browser_compatibility
+    if (window.nullableTrustedTypes != null) {
```

Comment:
I updated this code path to use the TrustedHTML setter, if possible.

Reply:
Ah no need! Just noticed the `getTrustedTypesPolicy` extension.